### ISSUE TITLE
ErrorHandler: TypeError: cellWidth.replace is not a function     at index.js:385:105

### ISSUE THREAD

**Issue by wandaatest** [Aymkdn/html-to-pdfmake#210]
my project only works on` "html-to-pdfmake":"2.5.2",`
plz help me to fix the problem, is there any reason cause this? table width?
i have no idea about this..
"html-to-pdfmake": "2.5.5",
"html-to-pdfmake": "2.5.6",
it also shows this consloe error
`ErrorHandler: TypeError: cellWidth.replace is not a function     at index.js:385:105`

**Comment by Aymkdn** [comment]
Please, provide the HTML code that you transform.

**Comment by soeren-krueger** [comment]
Reproduction:

```
htmlToPdfmake(
  `<table style="width: 100%;">
    <tr>
      <td style="width: 160px;"></td>
    </tr>
  </table>`,
  { tableAutoSize: true }
);
```

**Comment by Aymkdn** [comment]
Thanks for the example. I'll have a look at it.

**Comment by Aymkdn** [comment]
It's now fixed with v2.5.7
